<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettingsWidget</class>
 <widget class="QWidget" name="GeneralSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::FieldsStayAtSizeHint</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="languageLabel">
     <property name="text">
      <string>Language:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="languages">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="showAs">
     <property name="text">
      <string>Show application icon in menu bar</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="showWhereLabel">
     <property name="text">
      <string>Show:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="showWhere"/>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="launch">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Launch application with media players</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="notifications">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Show desktop notifications</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="lastRadio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Remember last radio station and tune in on start up</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="sendCrashReports">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Send crash reports to Last.fm</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="updates">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Check for updates automatically</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="mediaKeys">
     <property name="text">
      <string>Enable media keys</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
